crates/kobold/src/lib.rs

```diff
@@ -245,6 +245,7 @@
 //! trunk serve
 //! ```
 
+#![feature(toowned_clone_into)]
```

Comment:
```suggestion
```

[`clone_into` has been stable since 1.63](https://doc.rust-lang.org/std/borrow/trait.ToOwned.html#method.clone_into), `rustup update` maybe? :)